src/components/BookList.jsx

```diff
@@ -0,0 +1,48 @@
+import { useState, useEffect } from 'react';
+import Book from './Book';
+import { v4 as uuidv4 } from "uuid";
+import CreateABookForm from './CreateABookForm';
+
+const BookList = () => {
+  // Retrieve stored books from local storage
+  const getBooks = () => {
+    const temp = localStorage.getItem("books");
+    const storedBooks = JSON.parse(temp);
+    return storedBooks || [];
+  };
+  const [books, setBooks] = useState(getBooks());
+
+  // Update local storage when books state changes
+  useEffect(() => {
+    const temp = JSON.stringify(books);
+    localStorage.setItem("books", temp);
+  }, [books]);
+
+  // Add Book
+  const handleAddBookItem = (title, author) => {
```

Comment:
- [ ] Great job working on the project so far but I think you missunderstand some of the requirements since you should render a list of static books but after running your project I could see that there is no list rendered so kindly render a static list Contains a list to renders individual books 🤩🔥🙏🏻

## The  button for removing books  wouldn't remove the books yet, this is just a placeholder for now 🙏🏻
![image](https://github.com/DebyGrey/bookstore/assets/58553711/44a965a8-791a-4364-9660-f768776fc3de)

Reply:
@Reem-lab I am having a hard time understanding what you want me to do here. Did you want me to deactivate the delete button functionality?

Reply:
@Reem-lab I just did that, thank you.